src/CMakeLists.txt

```diff
@@ -472,6 +475,8 @@ set(minetest_SRCS
 	sky.cpp
 	tile.cpp
 	wieldmesh.cpp
+	network/clientopcodes.cpp
+	network/toclientpacket.cpp
```

Comment:
Wouldn't it make more sense to move these up to the top to the other files you added?